### DISCUSSION TITLE
Documentation: where is the zstyle system?

### DISCUSSION THREAD

**Discussion by kslstn** [marlonrichert/zsh-autocomplete#206] [Q&A] [answered]
> The behavior of zsh-autocomplete can be customized through the zstyle system.

I'd love to know where I should add the lines mentioned in the section :)

**Comment by marlonrichert** [comment, marked as answer]
Just add any of the `zstyle` lines from the Readme to your `~/.zshrc` file to change your settings.

I've now added a note about it [to the Readme](https://github.com/marlonrichert/zsh-autocomplete/blob/master/README.md#preferences).
